diff --git a/node_modules/torrent-stream/index.js b/node_modules/torrent-stream/index.js
index 77ca861..349c4a8 100644
--- a/node_modules/torrent-stream/index.js
+++ b/node_modules/torrent-stream/index.js
@@ -33,7 +33,7 @@ var BAD_PIECE_STRIKES_DURATION = 120000 // 2 minutes
 var RECHOKE_INTERVAL = 10000
 var RECHOKE_OPTIMISTIC_DURATION = 2
 
-var TMP = fs.existsSync('/tmp') ? '/tmp' : os.tmpDir()
+var TMP = fs.existsSync('/tmp') ? '/tmp' : os.tmpdir()
 
 var noop = function () {}
 
